tests/internal/test_remoteconfig.py

```diff
@@ -30,3 +111,22 @@ def test_remote_config_forksafe():
         assert RemoteConfig._worker is not None
         assert RemoteConfig._worker is not parent_worker
         exit(0)
+
+
+@mock.patch.object(RemoteConfigClient, "_send_request")
+def test_remote_configuration(mock_send_request):
+    class Callback:
+        features = {}
+
+        def _reload_features(self, metadata, features):
+            self.features = features
+
+    callback = Callback()
+
+    mock_send_request.return_value = get_mock_encoded_msg(b'{"asm":{"enabled":true}}')
+    rc = RemoteConfig()
+    rc.register(ASM_FEATURES_PRODUCT, callback._reload_features)
+    sleep(3)
+    mock_send_request.assert_called_once()
+    print(callback.features)
```

Comment:
Are the `sleep` and the `print` required for the test? Maybe we can at least reduce the sleep time?

Reply:
I tried with 1sec and RC wasn't working, 2sec random success and 3sec was the minimun value with 100% success rate

Reply:
The RCM polling interval can be controlled with `DD_REMOTECONFIG_POLL_SECONDS`. Perhaps setting that to e.g. 0.5 before registering might help reduce the sleep time?

Reply:
:clap: good idea @P403n1x87